addons/hr_expense/static/src/views/list.js

```diff
@@ -44,11 +44,30 @@ export class ExpenseListController extends ListController {
         return this.is_account_invoicing && records.length && records.every(record => record.data.state === 'approve') && this.isExpenseSheet;
     }
 
+    displayPayment() {
+        const records = this.model.root.selection;
+        return this.is_account_invoicing && records.length && records.every(record => record.data.state === 'post' && record.data.payment_state === 'not_paid') && this.isExpenseSheet;
+    }
```

Comment:
Not really related to this pr but `is_account_invoicing` should be in camelCase not in snake_case